adopt-a-dog/styles/styles.css

```diff
@@ -0,0 +1,246 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Arial', sans-serif;
+  font-size: 14px;
+  color: grey;
+  kerning: 2px;
+}
+
+.page-container {
+  height: 100%;
+  margin: 0 auto;
+  }
+
+.header {
+  background-image: url(../images/christopher-harris-57366-unsplash.jpg);
+  background-size: cover;
+  width: 100%;
+  height: 100%;
+  color: white;
+}
+
+.tag-line {
+  color: #8fc518;
+  font-weight: bold;
+  font-size: 25px;
+  font-style: italic;
+  margin-left: 3rem;
+  padding-left: 2rem;
+  font-family: serif;
+  padding-bottom: 5rem;
+  margin-bottom: 6rem;
+}
+
+.checkout {
+  text-align: right;
+  padding: 1rem;
+}
+
+.logo {
+  height: auto;
+  width: 35%;
+  margin-left: 3rem;
+  padding-left: 2rem;
+}
+
+.nav {
+  background-color: #f0f0f0;
+  padding: .50rem;
+}
+
+.nav-li {
+  display: inline;
+  text-decoration: none;
+  font-weight: bold;
+  color: #2F425F;
+}
+
+h2, h3, h4, h5 {
+  color: #2F425F;
+  font-family: serif;
+}
+
+h1 {
+  color: #2F425F;
+  font-family: serif;
+  margin-left: 1rem;
+
+}
+
+li {
+  list-style-type: none;
+  display: inline;
+}
+
+li:after {
+  content: " / ";
+  padding: 1rem;
+
+}
+li:last-child:after {
+    content: " ";
+  }
```

Comment:
Keep a close eye on your formatting. Delete extra lines (like line 70) and make sure there is a line seperating all of your css selectors (like line 77, but could use one above line 83) also make sure that the closing brackets are left aligned and everything inside is indented one level.